plugins/content-manager/src/main/java/com/wazuh/contentmanager/model/rest/Credentials.java

```diff
@@ -0,0 +1,45 @@
+package com.wazuh.contentmanager.model.rest;
+
+public class Credentials {
+    private static Credentials instance;
```

Comment:
Make it volatile since if the instance suffers changes it will help synchronize its usages
https://www.baeldung.com/java-volatile